### ISSUE TITLE
Remove LICENSE, license is a capitalist concept

### ISSUE THREAD

**Issue by mmankows** [jokteur/python_communism#10]
as per title

**Comment by jokteur** [comment]
Solved

**Comment by jokteur** [comment]
This is why the unlicense license has been chosen. I hope it is an acceptable compromise.

**Comment by jabezborja** [comment]
Agree with this. THIS IS **OUR** GITHUB REPO.

![image](https://user-images.githubusercontent.com/64759159/149646087-b64e9f3b-cb7a-46e5-ae19-4b308fddb5fd.png)

**Comment by jokteur** [comment]
Yes, of course I agree, it is **_OUR_** repo. However I am the self proclaimed maintainer. The Unlicence stays.

**Comment by igoro00** [comment]
> Yes, of course I agree, it is **_OUR_** repo. However I am the self proclaimed maintainer. The Unlicence stays.

Some people are just more equal than others.
